### ISSUE TITLE
License update, anyway you could relicense this as Apache/MIT? 

### ISSUE THREAD

**Issue by visch** [aaronsteers/tap-ldap#4]
Not a huge deal, as I'm going to contribute back no matter what but it saves me the worry of AGPL licenses

**Comment by visch** [comment]
Never mind, I took a quick dive into the code and it's probably going to be faster to just use the sdk

**Comment by visch** [comment]
> Never mind, I took a quick dive into the code and it's probably going to be faster to just use the sdk

and instead I think csvde is going to win this battle for today :/

**Comment by aaronsteers** [comment]
@visch - Sorry, I just saw this today. Yes, for any code I am sole owner on, I hereby approve re-license as Apache/MIT. Since this was pre-SDK days, I was starting from an AGPLv3 skeleton. I can't relicense that code, but today it would be easy enough to rebuild on the SDK - as you point out above.

TL;DR;: All of the LDAP-specific code can be relicensed and the remainder is not needed if building on the SDK.
